docs/example.md

```diff
@@ -0,0 +1,293 @@
+## Blender addon
+
+![](./assets/blender_example_addon.png)
+
+_Blender addon using Pyscriptpacker for packing into a zip file_
+
+Here is a simple example, using Pyscriptpacker to pack this blender [:octicons-file-code-24: example addon](https://github.com/3dninjas/pyscriptpacker). 
+
+Packer command line:
+
+``` console
+$ pyscriptpacker -m ./unpacked/__init__.py addon ./unpacked ./example_addon/__init__.py
+```
+
+??? example "Result file"
```

Comment:
I dont think it makes sense to paste the result file here. How to use the packed file, that is more important.